app/partials/class-list.html

```diff
@@ -1,7 +1,8 @@
 <div>
 <h2>Concept Class Management</h2>
+	<h3>{{(classAdded.length > 0) && (classAdded != ' ') ? classAdded + ' has been added' : ''}}</h3>
```

Comment:
You typically put conditions in ng-show so that the h3 element is displayed or hidden completely